go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/nats-io/nats-server/v2
 
 require (
-	github.com/nats-io/jwt v0.2.14
+	github.com/nats-io/jwt v0.2.15-0.20190917180649-91932864a4fd
```

Comment:
I am assuming this is temporary waiting for a JWT release, right? Also, will need to tidy because the sum start to have old versions of things it looks like (but they may be required, don't know).

Reply:
Tes will do JWT release now and update this PR.